xdg-desktop-portal-wlr needs to call a program to choose screencast output

**Describe the bug**
xdg-desktop-portal-wlr calls a program (by default ``slurp``) to choose which output to screencast since v0.3.0. This program is not in xdpw's path so it can't be called. This breaks screen sharing on sway.

**To Reproduce**

Try to screencast with any program that supports it. For the following steps, I'll be using Chromium with the ``#enable-webrtc-pipewire-capturer`` flag set to ``Enabled``.

My system is built with the nixos-unstable channel (currently on 7cb76200088f45cd24a9aa67fd2f9657943d78a4 as of my tests).
Make sure you have ``sway``, ``xdg.portal`` and ``pipewire`` enabled with ``xdg-desktop-portal-wlr`` in ``xdg.portal.extraPortals``.
Make sure you applied the workaround from https://github.com/NixOS/nixpkgs/issues/119445#issuecomment-820507505

Steps to reproduce the behavior:
1. Open the [gUM test page](https://mozilla.github.io/webrtc-landing/gum_test.html)
2. Click on ``Screen capture``
3. The ``Choose what to share menu`` opens, click in the middle left of this window, it will select the blank preview of the screencast.
4. Click on the ``Share`` button.

When screencasting, a blank screen will appear on the page
In a terminal, run ``journalctl --user -e -u xdg-desktop-portal-wlr``, you will see that xdpw tried to run ``slurp`` but got a command not found (see example).
```
May 04 11:40:58 mercury xdg-desktop-portal-wlr[3095]: 2021/05/04 11:40:58 [ERROR] - wlroots: no output found
May 04 11:40:59 mercury xdg-desktop-portal-wlr[3345]: /bin/sh: slurp: command not found
```

**Expected behavior**
By default, slurp will be launched and you will be able to pick which monitor output to screen cast. After that, screencasting will work.
It is also possible to change the program that is launched to pick the output to screencast in xdpw's config file.

**Notify maintainers**
@minijackson 

**Metadata**
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.32, NixOS, 21.05pre286880.7cb76200088 (Okapi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.4pre20210503_6d2553a`
 - channels(root): `"nixos-21.05pre286880.7cb76200088"`
 - channels(nicolas): `""`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

--- a/pkgs/development/libraries/xdg-desktop-portal-wlr/default.nix
+++ b/pkgs/development/libraries/xdg-desktop-portal-wlr/default.nix
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchFromGitHub, makeWrapper
 , meson, ninja, pkg-config, wayland-protocols
-, pipewire, wayland, systemd, libdrm, iniparser, scdoc, grim }:
+, pipewire, wayland, systemd, libdrm, iniparser, scdoc, grim, slurp }:
 
 stdenv.mkDerivation rec {
   pname = "xdg-desktop-portal-wlr";
@@ -21,7 +21,7 @@ stdenv.mkDerivation rec {
   ];
 
   postInstall = ''
-    wrapProgram $out/libexec/xdg-desktop-portal-wlr --prefix PATH ":" ${lib.makeBinPath [ grim ]}
+    wrapProgram $out/libexec/xdg-desktop-portal-wlr --prefix PATH ":" ${lib.makeBinPath [ grim slurp ]}
   '';
 
   meta = with lib; {

